man/fi_domain.3.md

```diff
@@ -108,32 +108,33 @@ with a domain.
 
 ```c
 struct fi_domain_attr {
-	struct fid_domain     *domain;
-	char                  *name;
-	enum fi_threading     threading;
-	enum fi_progress      control_progress;
-	enum fi_progress      data_progress;
-	enum fi_resource_mgmt resource_mgmt;
-	enum fi_av_type       av_type;
-	int                   mr_mode;
-	size_t                mr_key_size;
-	size_t                cq_data_size;
-	size_t                cq_cnt;
-	size_t                ep_cnt;
-	size_t                tx_ctx_cnt;
-	size_t                rx_ctx_cnt;
-	size_t                max_ep_tx_ctx;
-	size_t                max_ep_rx_ctx;
-	size_t                max_ep_stx_ctx;
-	size_t                max_ep_srx_ctx;
-	size_t                cntr_cnt;
-	size_t                mr_iov_limit;
-	uint64_t              caps;
-	uint64_t              mode;
-	uint8_t               *auth_key;
-	size_t                auth_key_size;
-	size_t                max_err_data;
-	size_t                mr_cnt;
+	struct fid_domain       *domain;
+	char                    *name;
+	enum fi_threading       threading;
+	enum fi_progress        control_progress;
+	enum fi_progress        data_progress;
+	enum fi_resource_mgmt   resource_mgmt;
+	enum fi_av_type         av_type;
+	int                     mr_mode;
+	size_t                  mr_key_size;
+	size_t                  cq_data_size;
+	size_t                  cq_cnt;
+	size_t                  ep_cnt;
+	size_t                  tx_ctx_cnt;
+	size_t                  rx_ctx_cnt;
+	size_t                  max_ep_tx_ctx;
+	size_t                  max_ep_rx_ctx;
+	size_t                  max_ep_stx_ctx;
+	size_t                  max_ep_srx_ctx;
+	size_t                  cntr_cnt;
+	size_t                  mr_iov_limit;
+	uint64_t                caps;
+	uint64_t                mode;
+	uint8_t                 *auth_key;
+	size_t                  auth_key_size;
+	size_t                  max_err_data;
+	size_t                  mr_cnt;
+	uint32_t		tclass;
```

Comment:
Something is off here.  This should have been a 1-line change, but it shows every line updated?  I think you added tabs before tclass.  Please use spaces in the markdown pages

Reply:
I still need to re-adjust it. The space changes were consistent with the original length of the tclass declaration (originally struct traffic_class), but not with how it is now (uint32_t)

Reply:
Done